Packages/com.llealloo.audiolink/Runtime/Scripts/ytdlpPlayer.cs

```diff
@@ -415,7 +465,17 @@ public override void OnInspectorGUI()
 #endif
             }
         }
-    }
 
+        /// <summary>
+        /// Draws some text in the same style as the [Header] attribute.
+        /// </summary>
+        /// <param name="header">the desired text to draw</param>
+        /// <param name="fontSizeDelta"></param>
+        public static void Header(string header, int fontSizeDelta = 0)
```

Comment:
this doesn't seem like something that needs to be made public

Reply:
That can change. This was just some helper methods I pulled over from a utility class.